src/functions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+require_once 'src/db_structures.php';
+//function that converts the dogs array into its individual fields from the db_functions
+function returnDogs(array $dog_data): string
```

Comment:
`displayDogs`? 

Many functions can `return`, so in general naming a function `returnSomething` is not particularly helpful to readers of your code.